src/ontology/cl-edit.owl

```diff
@@ -30104,6 +30107,36 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030011 "epithelial cell of
 AnnotationAssertion(rdfs:label obo:CL_4030011 "epithelial cell of proximal tubule segment 3")
 SubClassOf(obo:CL_4030011 obo:CL_0002306)
 
+# Class: obo:CL_4030012 (kidney loop of Henle short descending thin limb epithelial cell)
+
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/contributor> <http://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:33769951"^^xsd:string) obo:IAO_0000115 obo:CL_4030012 "Epithelial cell of the descending thin limb of the short loop (cortical) nephron limited to the outer medulla (mainly inner strip). It is known in some mammalian species that the short descending limb of the loop of Henle selectively expresses the serine protease Corin, the homeobox TF Uncx, and the urea channel Slc14a2."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030012 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030012 "2022-05-13T11:00:29Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:editor_note obo:CL_4030012 "TO DO: import UBERON:0005099 'short descending thin limb' and add subClass: part_of some UBERON:0005099"^^xsd:string)
```

Comment:
@bvarner-ebi I'd rather move this action item to a ticket, unless you're confident that you'll get to it soon, and that you'll remember to remove the editor note promptly ;-) But up to you really.